src/main/java/net/dv8tion/jda/core/entities/Category.java

```diff
@@ -102,4 +103,58 @@
      */
     @CheckReturnValue
     ChannelAction createVoiceChannel(String name);
+
+    /**
+     * Modifies the positional order of this Categories nested {@link #getTextChannels() TextChannels}.
+     * <br>This uses a specialized {@link net.dv8tion.jda.core.requests.RestAction RestAction} extension very similar to a
+     * {@link net.dv8tion.jda.core.requests.restaction.order.ChannelOrderAction ChannelOrderAction} but only controlling
```

Comment:
`This uses an extension of {@link net.dv8tion.jda.core.requests.restaction.order.ChannelOrderAction ChannelOrderAction} specialized for ordering the channels of a Category.`